compiler/z/codegen/OMRMemoryReference.cpp

```diff
@@ -1298,6 +1298,8 @@ OMR::Z::MemoryReference::populateAddTree(TR::Node * subTree, TR::CodeGenerator *
 
    noteAllNodesWithRefCountNotOne(nodesBefore, subTree, comp);
 
+   // Note: The addressChild for the 2nd aladd of an arraycopy can be NULL.
+   // This implies that the integer child is to be used directly as the address.
```

Comment:
This comment seems a bit out of place, as it does not relate to the code here.  I think, in general, populateMemoryReference(..) will just try to set _baseRegister over _indexRegister.   Where a typical base address node is an aconst NULL, the usual index subtree will be evaluated to fill _baseRegister and _offset instead.

Reply:
Agreed. Deleted comment.